examples/move/basics/sources/clock.move

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module basics::clock {
+    use sui::{clock::Clock, event};
+
+    public struct TimeEvent has copy, drop, store {
+        timestamp_ms: u64,
+    }
+
+    entry fun access(clock: &Clock) {
+        event::emit(TimeEvent { timestamp_ms: clock.timestamp_ms() });
+    }
+}
+
+module basics::clock_tests {
+    #[test]
+    fun creating_a_clock_and_incrementing_it() {
```

Comment:
Note that this example already exists, here:

https://github.com/MystenLabs/sui/blob/c6ba40b65185defcaef45c6a61ba06f9ba2458e9/crates/sui-framework/packages/sui-framework/tests/clock_tests.move#L9